src/helpers/index.tsx

```diff
@@ -306,6 +308,53 @@ export const handleContractError = (e: any) => {
   if (EnvHelper.env.NODE_ENV !== "production") console.warn("caught error in slices; usually network related", e);
 };
 
+/**
+ * Converts a supported networkId to its hexadecimal value as a string
+ */
+export const networkIdToHexString = (id: NetworkId) => "0x" + id.toString(16);
```

Comment:
we already have `idToHexString` in `helpers/NetworkHelper.ts`

Reply:
Should I remove this one and just use the NetworkHelper version then? Only put it here because it seemed like an appropriate place for a generic helper function like this

Reply:
IMO yea I would remove this one & use that one just so we don't repeat the code two ways.

Reply:
Done